src/wp-content/themes/twentytwentyone/inc/template-functions.php

```diff
@@ -229,7 +229,7 @@ function twenty_twenty_one_change_calendar_nav_arrows( $calendar_output ) {
  * @since Twenty Twenty-One 1.0
  *
  * @param string $type Whether to return CSS for the "front-end", "block-editor", or "classic-editor".
- * @return string
+ * @return string CSS styles for non-Latin languages based on the site locale.
  */
 function twenty_twenty_one_get_non_latin_css( $type = 'front-end' ) {
```

Comment:
This function is calling `twenty_twenty_one_generate_css()` which also lacks a `@return` description.